components/src/core/InfoListItem/InfoListItem.test.tsx

```diff
@@ -41,10 +39,11 @@ describe('InfoListItem', () => {
         wrapper = shallow(<InfoListItem hidePadding title="Test" />);
         expect(wrapper.find(PersonIcon).length).toEqual(0);
     });
-
+/* No longer using in-line props to modify styling.
```

Comment:
Can we find another way to test this, or add another test of some sort?

Reply:
Absolutely, I'm going to fix this test.  I'll re-request review when the tests are passing.